lib-api/src/main/java/org/ergoplatform/appkit/BlockchainContext.java

```diff
@@ -67,5 +68,8 @@
 
     /** Get unspent boxes protected by given ergo tree template */
     List<InputBox> getUnspentBoxesForErgoTreeTemplate(ErgoTreeTemplate template);
+
+    Optional<InputBox> getBoxByIdFromExplorer(ErgoId boxId);
+
```

Comment:
Why `InputBox[] getBoxesById(String... boxIds) throws ErgoClientException;` is not enough?

Reply:
`InputBox[] getBoxesById(String... boxIds) throws ErgoClientException;` uses node's `utxo/byId/{boxId}` returning `ErgoTransactionOutput` and I need to call explorer's `transactions/boxes/{id}` returning `TranscationOutput` because I need it's `spentTransactionId`id of the tx where this box was used as input.